pennylane/tape/tapes/tape.py

```diff
@@ -22,7 +22,6 @@
 
 import pennylane as qml
 
-from pennylane.operation import State
```

Comment:
How come this has been removed?

Reply:
It was grayed out, let me check.

Reply:
Oh, it is used only once in the file, but with `qml.operation.State`. The `JacobianTape` uses it without the import path, which is why it's imported there. Do we have a convention?

Reply:
Good to leave it out :) The fewer `from ... import ...` statements we have, the less the chance of circular dependencies